frontend/src/scenes/session-recordings/player/PlayerMeta.tsx

```diff
@@ -58,6 +58,25 @@ function URLOrScreen({ lastUrl }: { lastUrl: string | undefined }): JSX.Element
     )
 }
 
+function PlayerWarningsRow(): JSX.Element | null {
+    const { messageTooLargeWarnings } = useValues(sessionRecordingPlayerLogic)
+    return messageTooLargeWarnings.length ? (
+        <div>
+            <LemonBanner
+                type="error"
+                action={{
+                    children: 'Learn more',
+                    to: 'https://posthog.com/docs/session-replay/troubleshooting#message-too-large-warning',
+                    targetBlank: true,
+                }}
+            >
+                This session recording had {messageTooLargeWarnings.length} messages that were too large to be captured.{' '}
```

Comment:
Not sure if the wording `messages` here is ideal, snapshots is probably also very technical, but I wonder if they will understand what message here means

Reply:
yeah good point... something vague like "data" is probably the way to go 👍